gpg/decrypt.go

```diff
@@ -18,21 +20,70 @@ func NewGPGHelper(decrypter StringDecrypter) *GPGHelper {
 	}
 }
 
+func gpg_major_version() (int, error) {
+	cmdName := "gpg"
+	cmdArgs := []string{"--version"}
+
+	var cmdOut []byte
+	var err error
+
+	if cmdOut, err = exec.Command(cmdName, cmdArgs...).Output(); err != nil {
+		fmt.Println("There was an error running gpg --version: ", err)
+		return -1, err
+	}
+	gpgvers := string(cmdOut)
+
+	// gpg1 --version output should look similar to
+	//
+	//	gpg (GnuPG) 1.4.20
+	//	Copyright (C) 2015 Free Software Foundation, Inc.
+	//...
+
+	// gpg2 --version
+	//	gpg (GnuPG) 2.2.5
+	//	libgcrypt 1.8.2
+	//	Copyright (C) 2018 Free Software Foundation, Inc.
+
+	for i := 0; i < len(gpgvers); i++ {
+		if gpgvers[i] == '1' {
+			return 1, nil
+		}
+		if gpgvers[i] == '2' {
+			return 2, nil
+		}
+	}
+	return -1, errors.New("Could not determine gpg major version")
+}
+
 // Decrypt GPG keys
 func Decrypt(key string) (string, error) {
 
 	var cmd exec.Cmd
 	var output bytes.Buffer
+	var gpgvers int
 
 	gpgCmd, err := exec.LookPath("gpg")
 
 	if err != nil {
 		return "", err
 	}
 
-	cmd.Path = gpgCmd
-	cmd.Args = []string{"--decrypt", "--quiet"}
+	gpgvers, err = gpg_major_version()
+	if err != nil {
+		fmt.Println(err)
+		fmt.Println("Due to error determining gpg version, defaulting to gpg vers 1 options")
```

Comment:
I'd like to keep the UI consistent here. Can you import logrus and make these `log.Warn` messages?